operator/v1/types_ingress.go

```diff
@@ -1884,6 +1884,22 @@ type IngressControllerTuningOptions struct {
 	// +optional
 	ConnectTimeout *metav1.Duration `json:"connectTimeout,omitempty"`
 
+	// httpKeepAliveTimeout defines the maximum allowed time to wait for
+	// a new HTTP request to appear.
```

Comment:
This refers to waiting for a new HTTP request to appear on an idle connection that is being considered for closure, right?  If that is the purpose, or one of the purposes of this timeout, it should be mentioned.